packages/components/src/button/index.native.js

```diff
@@ -106,6 +107,7 @@ export function Button( props ) {
 			accessibilityRole={ 'button' }
 			accessibilityHint={ hint }
 			onPress={ onClick }
+			onLongPress={ onLongPress }
```

Comment:
Minor non-blocker: The web version of the button components passes down all the props not explicitly used: 		https://github.com/ceyhun/gutenberg/blob/f49806990a8d9219d04c7dbb9cfefb2560f6823d/packages/components/src/button/index.js#L56

Would it make sense to do something equivalent on mobile and simply pass down props to TouchableOpacity without explicitly referring onLongPress?

Reply:
Maybe there could be some props passed in that get picked up upper in the tree that we may or may not want to pass to TouchableOpacity directly. 
In that case it could be better to be explicit about it.